File: Lab8/ModifiedMVCExample/SourceCode/View/StudentView.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package View;

import Model.Student;

/**
 *
 * @author Ghani
 */
public class StudentView {
    public void showStudentDetails(Student std){
        System.out.println("---------- STUDENT INFO -----------");
        System.out.println("Name: "+std.getName());
        System.out.println("Reg No: "+std.getRegNo());
        System.out.println("Semister: "+std.getSemister());
        System.out.println("GPA : "+std.getGpa());
    }
}

File: Lab8/MVCInPython/InPython/view/student_view.py
# view/student_view.py

class StudentView:
    @staticmethod
    def show_student_details(student):
        print("---------- STUDENT INFO -----------")
        print(f"Name: {student.get_name()}")
        print(f"Reg No: {student.get_reg_no()}")
        print(f"Semester: {student.get_semester()}")
        print(f"GPA: {student.get_gpa()}")